### PR TITLE
Refactor/ak game page styling

### PR DESCRIPTION
**Pull request by annamkummer**
# What (if any) features are you implementing?
- N/A
# What (if anything) did you refactor?
- A few small updates to the game page styling
# Were there any issues that arose?
-  N/A
# Is there anything that you need from your teammate?
-  N/A
# Any other comments, questions, or concerns?
-  N/A

### CODE REVIEW

File: src/css/Game.css [hunk 1]
```diff
@@ -36,10 +36,17 @@
   transform: scale(1.05);
 }
 
-.next-btn:active {
+.next-btn:disabled {
```
Comment (jkiernan12) [inline src/css/Game.css:39]: Good call on this!

File: src/css/Nav.css [hunk 2]
```diff
@@ -25,6 +25,11 @@
   font-weight: bold;
 }
 
+.active {
+  cursor: default;
+  text-decoration: underline wavy #e2931d;
```
Comment (jkiernan12) [inline src/css/Nav.css:30]: Looks cool :)

**Review by jkiernan12 — APPROVED** [review]

File: src/css/Game.css (hunk 1 above)
Comment (NEwertKrocker) [reply src/css/Game.css:39]: Yeah, thanks for this.

**Review by NEwertKrocker — APPROVED** [review]
Excellent! Thanks for deleting some of that old, commented-out code, as well.

**Pull request merged**